common/footer.html

```diff
@@ -32,127 +32,135 @@
             <div class="footer-left">
               <a
-                href="https://www.freecodecamp.org/news/what-is-a-proxy-server-in-english-please/"
-                >What is a Proxy Server?</a
+                href="https://www.freecodecamp.org/news/responsive-web-design-how-to-make-a-website-look-good-on-phones-and-tablets/"
+                >Responsive Web Design</a
               >
               <a
-                href="https://www.freecodecamp.org/news/the-cat-command-in-linux-concatenation-explained-with-bash-examples/"
-                >Cat Command in Linux</a
+                href="https://www.freecodecamp.org/news/svg-basics-what-are-scalable-vector-graphics-and-how-do-you-use-them/"
+                >What Is an SVG File?</a
               >
               <a
-                href="https://www.freecodecamp.org/news/how-to-add-an-image-url-to-your-div/"
-                >CSS Background Image</a
+                href="https://www.freecodecamp.org/news/pdf-password-remover-guide-how-to-remove-password-protection-from-a-pdf/"
+                >PDF Password Remover</a
               >
               <a
-                href="https://www.freecodecamp.org/news/html-background-color-tutorial-how-to-change-a-div-background-color-explained-with-code-examples/"
-                >HTML Background Color</a
+                href="https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/"
+                >What Is a PDF?</a
               >
-              <a href="https://www.freecodecamp.org/news/comments-in-css/"
-                >CSS Comment Example</a
+              <a
+                href="https://www.freecodecamp.org/news/what-is-python-used-for-10-coding-uses-for-the-python-programming-language/"
+                >What Is Python?</a
               >
             </div>
 
             <div class="footer-right">
               <a
-                href="https://www.freecodecamp.org/news/arrow-function-javascript-tutorial-how-to-declare-a-js-function-with-the-new-es6-syntax/"
-                >Arrow Function JavaScript</a
+                href="https://www.freecodecamp.org/news/what-is-tls-transport-layer-security-encryption-explained-in-plain-english/"
+                >What Is TLS?</a
               >
               <a
-                href="https://www.freecodecamp.org/news/how-to-remove-duplicates-in-excel-delete-duplicate-rows-with-a-few-clicks/"
-                >Remove Duplicates in Excel</a
+                href="https://www.freecodecamp.org/news/what-is-a-lan-local-area-network-explained-in-plain-english/"
+                >What Is a LAN?</a
               >
+
               <a
-                href="https://www.freecodecamp.org/news/what-is-dllhost-exe-and-com-surrogate-in-windows-task-manager-solved/"
-                >dllhost.exe COM Surrogate</a
+                href="https://www.freecodecamp.org/news/what-is-npm-a-node-package-manager-tutorial-for-beginners/"
+                >What is npm?</a
```

Comment:
Just noticed this on /news, but maybe this should be capitalized like the other links:

What Is npm?